learn/safe-tools/safe-protocol-kit-safe-core-sdk/README.md

```diff
@@ -1,7 +1,281 @@
+
+Safe Factory is used to create a new Safe. While Safe class represents an instance of a  particular Safe account.
+
+```tsx
+import { SafeFactory } from '@safe-global/safe-core-sdk'
+
+const safeFactory = await SafeFactory.create({ ethAdapterOwner1 })
+```
+
+### Deploy a Safe
+
+```tsx
+import { SafeAccountConfig } from '@safe-global/safe-core-sdk'
+
+const safeAccountConfig = {
+  owners: [
+	  owner1Signer.getAddress(),
+		owner2Signer.getAddress(),
+ 		owner3Signer.getAddress()
+	],
+  threshold: 2,
+  // ... (optional params)
+}
+
+// this safe is tied to owner 1 because the factory was initialized with
+// an adapter that had owner 1 as the signer
+const safeSdkOwner1 = await safeFactory.deploySafe({ safeAccountConfig });
+
+console.log("Your Safe has been deployed:")
+console.log(`https://goerli.etherscan.io/address/${safeSdk.getAddress()}`))
+```
+
+Calling the method `deploySafe`will deploy the desired Safe and return a Safe Core SDK initialized instance ready to be used. Check the [API Reference](https://github.com/safe-global/safe-core-sdk/tree/main/packages/safe-core-sdk#deploysafe) for more details on additional configuration parameters and callbacks.
+
+### Send ETH to The Safe
+
+We will send some ETH to this Safe (treasury). Owner 1 will deposit 0.1 Goerli ETH to this Safe from our personal account following the [instructions in Quickstart](https://docs.gnosis-safe.io/learn/quickstart).
+
+```tsx
+const treasury = safeSdk.getAddress()
+
+const treasuryAmount = ethers.utils.parseUnits("0.1", 'ether').toHexString()
+
+const params = [{
+  to: treasury,
+  value: treasuryAmount
+}]
+
+const tx = await owner1Signer.sendTransaction(transactionParameters);
+
+console.log("Fundraising.")
+console.log(`Deposit Transaction: https://goerli.etherscan.io/tx/${tx.hash}`))
+```
+
+## Making a Transaction from a Safe
+
+Owner 1 will sign and propose a transaction to send 0.05 ETH out of the Safe. Then, owner 2 will add their own proposal and execute the transaction since it meets the 2 of 3 thresholds.
+
+At a high level, making a transaction from the Safe requires the following steps:
+
+### Overview
+
+1. Owner 1 creates a transaction
+    1. Define the amount, destination, and any additional data to include in the transaction
+    2. Deterministically generate a hash
+2. Owner 1 proposes a transaction
+    1. Performs an off-chain signature of the transaction before proposing
+    2. Submits it to the Safe Service
+    3. Adds the signature to get 1 of 2 threshold
+3. Owner 2  gets pending transactions from the Safe service
+4. Owner 2  confirms the transaction
+    1. Perform off-chain signature
+    2. Submits to service
+5. Anyone executes the transaction (Owner 1 in this example)
+    1. On-chain transaction
+    2. The only step that is submitted to the blockchain
+    3. Gas is paid
```

Comment:
```suggestion
5. Anyone can execute the transaction (Owner 1 in this example)
```
As these are comments rather than steps the user has to follow, I think it's better to add a short description about them (I would say 3 is irrelevant) in the corresponding section